### ISSUE TITLE
Aqara liquid Sensor

### ISSUE THREAD

**Issue by daniel-lo** [ioBroker/ioBroker.zigbee#353]
With new update lumi.sensor_wleak only reports "true".

**Comment by kirovilya** [comment]
This sensor regularly sends its status (as well as battery information). It should send the current status. If it wrong then try pairing this device again.

**Comment by kirovilya** [comment]
Hmm, check battery.

**Comment by daniel-lo** [comment]
100%

**Comment by kirovilya** [comment]
Very strange...

**Comment by daniel-lo** [comment]
I paired again but this time not with osram-socket (router) .. but with CC stick .. still the same. IMO something got broken with the latest update 0.11.3.

**Comment by kirovilya** [comment]
try to find file zigbee-shepherd-converters/devices.js 
from 361 line you will see:
```
    {
        zigbeeModel: ['lumi.sensor_wleak.aq1'],
        model: 'SJCGQ11LM',
        vendor: 'Xiaomi',
        description: 'Aqara water leak sensor',
        supports: 'water leak true/false',
        fromZigbee: [
            fz.xiaomi_battery_3v, fz.SJCGQ11LM_water_leak_iaszone,
            fz.SJCGQ11LM_water_leak_interval, fz.ignore_basic_change,
        ],
        toZigbee: [],
    },
```
remove `fz.SJCGQ11LM_water_leak_interval,` and restart adapter

**Comment by ptvoinfo** [comment]
@daniel-lo Could you please check the model number on the backside? Is it SJCGQ11LM?

**Comment by daniel-lo** [comment]
it's not there

`    {
        zigbeeModel: ['lumi.sensor_wleak.aq1'],
        model: 'SJCGQ11LM',
        vendor: 'Xiaomi',
        description: 'Aqara water leak sensor',
        supports: 'water leak true/false',
        fromZigbee: [
            fz.xiaomi_battery_3v, fz.SJCGQ11LM_water_leak_basic, fz.SJCGQ11LM_water_leak_iaszone,
            fz.ignore_basic_change,
        ],
        toZigbee: [],
    },
`

**Comment by daniel-lo** [comment]
@ptvoinfo 

Yes

**Comment by ptvoinfo** [comment]
@daniel-lo I have two water leak sensors too. I've just checked, both report the state correctly.

**Comment by daniel-lo** [comment]
@ptvoinfo 

Which adapter version are you on?

**Comment by ptvoinfo** [comment]
@daniel-lo I'm using the latest dev branch of Z2M, but it seems, the problem is related to a sensor.

**Comment by kirovilya** [comment]
> it's not there
> 
> ` { zigbeeModel: ['lumi.sensor_wleak.aq1'], model: 'SJCGQ11LM', vendor: 'Xiaomi', description: 'Aqara water leak sensor', supports: 'water leak true/false', fromZigbee: [ fz.xiaomi_battery_3v, fz.SJCGQ11LM_water_leak_basic, fz.SJCGQ11LM_water_leak_iaszone, fz.ignore_basic_change, ], toZigbee: [], },`

this one seems to be the wrong file. maybe it’s inside iobroker.zigbee/node-modules/zigbee-shepherd-converters/devices.js ?

**Comment by daniel-lo** [comment]
@ptvoinfo 

Not sure .. I wonder why the line is missing .. that I was supposed to delete and it would be a coincident that it seemingly happened with an update.

Maybe some other module needs to be updated?
The bottom line is that the sensor could be broken .. but I'm not there yet.

**Comment by daniel-lo** [comment]
![grafik](https://user-images.githubusercontent.com/4311558/64846216-58d7cc80-d60c-11e9-86ac-fc3c1b617234.png)

**Comment by kirovilya** [comment]
Try to check in /opt/iobroker/node_modules/iobroker.zigbee/node_modules/zigbee-shepherd-converters/devices.js
if exists

**Comment by daniel-lo** [comment]
Yes .. there I found it .. removed the line .. restarted adapter:

![grafik](https://user-images.githubusercontent.com/4311558/64846570-427e4080-d60d-11e9-987f-797ee4faabb0.png)

**Comment by kirovilya** [comment]
need to clear / delete state value to remove "true"

now always sends "true"?

**Comment by daniel-lo** [comment]
I deleted the object branch .. restarted adapter .. "true"

**Comment by ptvoinfo** [comment]
@daniel-lo, try to press the button on the sensor (single press on the top side, a led indicator should blink).

**Comment by daniel-lo** [comment]
@ptvoinfo .. I already tried pulling the battery .. but that didn't change anything

**Comment by kirovilya** [comment]
@daniel-lo Can you run the adapter in debug log mode, reproduce receive "true" and show logs?
![image](https://user-images.githubusercontent.com/8360230/64848120-37311200-d619-11e9-8028-8defd6146f1d.png)

**Comment by daniel-lo** [comment]
[iobroker.2019-09-13.log.txt](https://github.com/ioBroker/ioBroker.zigbee/files/3609107/iobroker.2019-09-13.log.txt)

**Comment by kirovilya** [comment]
it feels like the line from devices.js has not been deleted...
or the other devices.js file is used

**Comment by daniel-lo** [comment]
yes I rolled back the change ..wrong?

**Comment by kirovilya** [comment]
:) if you do not need "true" so far, delete the specified line.

perhaps we will remove it in the next release too

**Comment by daniel-lo** [comment]
New logfile without the specified line.

[iobroker.2019-09-13.log2.txt](https://github.com/ioBroker/ioBroker.zigbee/files/3609220/iobroker.2019-09-13.log2.txt)

**Comment by kirovilya** [comment]
now it is ok? no "true" messages

**Comment by daniel-lo** [comment]
still true .. always true

**Comment by daniel-lo** [comment]
First error I see related to that device .. of course I don't understand it:

zigbee.0 | 2019-09-14 08:14:21.304 | error | Zigbee publish to '0x00158d00024bfded', genBasic - 0 -  [{"attrId":16384},{"attrId":3}] - 1 failed with error Error: AF data  request fails, status code: 240. MAC transaction expired.
-- | -- | -- | --

**Comment by kirovilya** [comment]
You tried to change some state of this device and the adapter tried to send a command to the device. And these devices do not know how to receive commands.
In the future I will correct and the command will not be sent.

But you should not change read-only states :)

**Comment by daniel-lo** [comment]
tbh .. I didn't do anything with the zigbee adapter .. nor with any devices/states.

**Comment by daniel-lo** [comment]
In order to find out whether the sensor is broken or not I bought myself a Mi Control Hub and in fact the sensor is good .. the Mi App records water events properly.

**Comment by daniel-lo** [comment]
Interesting .. I had problems repairing the water sensor with my Osram socket .. so I repaired the Osram Socket with the CC-Stick and tried again. Didn't get that much easier but evenutally it worked. The divice didn't appear while I was pairing it .. it appeared while I was pairing the Aqara Cube again.
Now the Water sensor is false again and I tried a test alarm and that worked too. While the problem persisted .. othere devices on the Osram socket worked fine - so I don't really know what didn't work i.e. what made the Water Sensor report false all the time.
For now I could close the ticket .. because there is currently no problem. Nevertheless everthing remains to be a mystery for me. Possibly adapter updates did confuse things.
